lib/spoom/cli/bump.rb

```diff
@@ -19,13 +19,15 @@ class Bump < Thor
         desc: "Change files to this strictness"
       option :force, type: :boolean, default: false, aliases: :f,
         desc: "Change strictness without type checking"
+      option :encoding, type: :string, desc: "Use specific encoding when reading and writing files"
```

Comment:
Can we not always read/write files using `Encoding::ASCII_8BIT`, which basically means raw binary?

This seems to work:

```ruby
# Setup the file
str = 'puts "À coûté 10€"'
# => "puts \"À coûté 10€\""
File.write("foo.rb", str.encode("ISO-8859-15"))

# Actual Spoom routine will look like this
new_str = File.read("foo.rb", encoding: Encoding::ASCII_8BIT)
new_str = "typed: true\n" + new_str
File.write("foo.rb", new_str)

# Assert file contents
File.read("foo.rb", encoding: "iso-8859-15" )
# => "typed: true\nputs \"\xC0 co\xFBt\xE9 10\xA4\""
File.read("foo.rb", encoding: "iso-8859-15" ).encode("UTF-8")
# => "typed: true\nputs \"À coûté 10€\""
```

Reply:
^ This is pure genius! ❤️